{
    "testsuites": {
        "testsuite": {
            "@name": "validation_framework.validators.tests.nuvla_operations.test_engine_reboot.TestEngineReboot-20221213091053",
            "@tests": "1",
            "@file": "validation_framework/validators/tests/nuvla_operations/test_engine_reboot.py",
            "@time": "179.914",
            "@timestamp": "2022-12-13T09:13:53",
            "@failures": "0",
            "@errors": "1",
            "@skipped": "0",
            "testcase": {
                "@classname": "validation_framework.validators.tests.nuvla_operations.test_engine_reboot.TestEngineReboot",
                "@name": "test_engine_reboot",
                "@time": "179.914",
                "error": {
                    "@type": "ConnectionResetError",
                    "@message": "[Errno 54] Connection reset by peer",
                    "#text": "Traceback (most recent call last):\n  File \"/Users/nacho/PycharmProjects/nuvlaedge/devel/validation/validation_framework/validators/tests/nuvla_operations/test_engine_reboot.py\", line 75, in test_engine_reboot\n    while self.engine_handler.device.is_reachable(silent=True):\n  File \"/Users/nacho/PycharmProjects/nuvlaedge/devel/validation/validation_framework/deployer/target_device/ssh_target.py\", line 52, in is_reachable\n    it_result: fabric.Result = self.run_command('hostname')\n  File \"/Users/nacho/PycharmProjects/nuvlaedge/devel/validation/validation_framework/deployer/target_device/ssh_target.py\", line 88, in run_command\n    return connection.run(command, env=envs, hide=True)\n  File \"<decorator-gen-3>\", line 2, in run\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/fabric/connection.py\", line 29, in opens\n    self.open()\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/fabric/connection.py\", line 636, in open\n    self.client.connect(**kwargs)\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/paramiko/client.py\", line 450, in connect\n    self._auth(\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/paramiko/client.py\", line 697, in _auth\n    self._transport.auth_publickey(username, key)\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/paramiko/transport.py\", line 1635, in auth_publickey\n    return self.auth_handler.wait_for_response(my_event)\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/paramiko/auth_handler.py\", line 245, in wait_for_response\n    raise e\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/paramiko/transport.py\", line 2110, in run\n    ptype, m = self.packetizer.read_message()\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/paramiko/packet.py\", line 459, in read_message\n    header = self.read_all(self.__block_size_in, check_rekey=True)\n  File \"/Users/nacho/Library/Caches/pypoetry/virtualenvs/validation-3EvBwfil-py3.10/lib/python3.10/site-packages/paramiko/packet.py\", line 301, in read_all\n    x = self.__socket.recv(n)\nConnectionResetError: [Errno 54] Connection reset by peer"
                },
                "system-out": "~/.nuvlaedge_validation/engines/2.4.3/docker-compose.bluetooth.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.gpu.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.modbus.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.network.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.security.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.usb.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.yml\nnohup docker-compose -p NuvlaEdge_Validator -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.bluetooth.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.gpu.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.modbus.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.network.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.security.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.usb.yml -f ~/.nuvlaedge_validation/engines/2.4.3/docker-compose.yml up -d\n{'location': 'job/8ec3897f-c0bf-43c7-a0e8-901b5bed6f4a',\n 'message': 'sending reboot request to NuvlaBox '\n            'nuvlabox/23ab3939-6edd-4f56-b7c5-77cd33670051 with async '\n            'job/8ec3897f-c0bf-43c7-a0e8-901b5bed6f4a',\n 'resource-id': 'nuvlabox/23ab3939-6edd-4f56-b7c5-77cd33670051',\n 'status': 202}"
            }
        }
    },
    "target_device": "rpi4.toml"
}